src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -459,10 +506,10 @@ class VirtualMachine(
             return when (node.type) {
                 is StackType -> {
                     val stackValue = StackValue(EmptyMObject, Stack())
-                    stackValue.style = stylesheet.getStyle(assignLHS.identifier, stackValue)
                     val initStructureIdent = variableNameGenerator.generateNameFromPrefix("stack")
                     dataStructureBoundaries[initStructureIdent] = TallBoundary()
                     stackValue.style = stylesheet.getStyle(assignLHS.identifier, stackValue)
+                    stackValue.animatedStyle = stylesheet.getAnimatedStyle(assignLHS.identifier, stackValue)
```

Comment:
Added an `animatedStyle` property to both `StackValue` and `ArrayValue`, which uses the `getAnimatedStyle` function to merge the animated styles